pages/contribute.tsx

```diff
@@ -19,12 +31,123 @@ function Contribute(): JSX.Element {
             site_name: 'open source at ACM at UCLA',
           }}
         />
-        <h1>
-          get started
-        </h1>
+        <div className='row spaced-row'>
+          <h1>get started </h1>
+          <a href='#good-first-issues'><button>good first issues</button></a>
+        </div>
+        <hr />
+        <h2>
+          what is open source?
+        </h2>
+        <p>
+          Open source software is code that is made freely available for <b>anyone</b> to
+          use or enhance! Here at ACM, we take pride in <b>all</b> of our projects being open-source
+          and open for anyone to work on.
+        </p>
+        <h2>
+          what makes open source possible?
+        </h2>
+        <p>
+          The main back bone of open source initiatives are tools that can make collaboration and sharing of code as
+          simple as possible. Git and <ELink link='https://github.com/'>Github</ELink> are two really popular tools
+          that do just that used by companies like Google, Meta, and more, and is what we use at ACM as well.
+        </p>
+        <h2>
+          how can i contribute?
+        </h2>
+        <p>
+          We&apos;ll quickly go over how we can use git/github to work on some open source projects, and if
```

Comment:
Where are you going over how to use git/GitHub? 

(also, you can probably split up the run-on sentence here)

Reply:
We quickly go over cloning, forking, branching, etc. But, it could probably be hashed out better (maybe in a content PR?)